docs/source/contributing/docs.rst

```diff
@@ -0,0 +1,76 @@
+====================
+Adding Documentation
+====================
+
+Building the documentation
+--------------------------
+
+When you clone the Manim repository from GitHub, you can access the
+``docs/`` folder which contains the necessary files to build the
+documentation.
+
+To build the docs locally, enter the ``docs/`` folder in a CLI and
+run ``make html``. The first time you build them, the process will
```

Comment:
This is dependent on OS. For windows you will have to run `./make.bat html`.

Reply:
Thanks for mentioning it! I already made the corresponding change.